Stellar-ledger.x

```diff
@@ -398,6 +398,13 @@ struct DiagnosticEvent
     ContractEvent event;
 };
 
+struct ContractMetrics
+{
+    ExtensionPoint ext;
+    int64 cpuInsnsConsumed;
```

Comment:
Should these be unsigned?

Reply:
Maybe. I don't think it matters since many of the [configs](https://github.com/stellar/stellar-xdr/blob/next/Stellar-contract-config-setting.x) are also i64 for max generality. I don't feel too strongly either way.

Reply:
The actual tracking values in core are unsigned, so I think it's safer to make these unsigned as well.